### DISCUSSION TITLE
analytics

### DISCUSSION THREAD

**Discussion by reopucino** [gimid/warga#22] [Ideas]
saya agak lupa, tapi setau saya udah ada rencana untuk ini (pernah liat roadmapnya)

Tapi biar ga lupa saya tulis ulang.

Buat analytics untuk tulisan.. Seharusnya kedepannya bisa di pakai untuk jadi sessuatu (kayak tau level tulisan yang suka di baca, dan bisa tracing klo perkembangan user pembaca itu kemana)

**Comment by igrir** [comment]
yang ditulis di projects roadmap itu analytics dari Google Analytics sih om. Buat analytics in-house yang bisa diliat oleh masing-masing user belum ada fiturnya, dan kayaknya agak gede, bakal saya taruh jauh di belakang (atau kalau ada funding? ehe ehehe 😉 )

**Reply by reopucino** [reply]
hu uh sepertinya lumayan repot. Saya sendiri aja masih buta sama analytic.. gpp yang penting mumpung inget dan berasa discussionya rame :D

**Reply by igrir** [reply]
mungkin nanti saya bakal kasih rekap bulanan di sini deh ya

**Reply by reopucino** [reply]
kayaknya ga perlu dah om.. soalnya bakalan repot sepertinya.
